src/Magnum/ImGuiIntegration/Test/ContextGLTest.cpp

```diff
@@ -148,6 +166,22 @@ struct ContextGLTest: GL::OpenGLTester {
     void updateCursor();
 
     void multipleContexts();
+
+    void drawSetup();
+    void drawTeardown();
+
+    void draw();
+    void drawTexture();
+    void drawScissor();
+    void drawVertexOffset();
+    void drawIndexOffset();
+
+    private:
+        PluginManager::Manager<Trade::AbstractImporter> _manager{"nonexistent"};
+        std::string _testDir;
```

Comment:
```suggestion
        PluginManager::Manager<Trade::AbstractImporter> _manager;
```

And the `_testDir` can be just ~~`SHADERS_TEST_DIR`~~ `IMGUIINTEGRATION_TEST_DIR` now.